Dockerfile

```diff
@@ -57,6 +54,7 @@ RUN apt-get update && \
         mysql-client \
         poppler-utils \
         sudo \
+        vim \
```

Comment:
That`s true, I didn`t thought about that..
But there is no editor installed so which one should be installed?
vi? or nano? Like as the default on many Linux distributions.